CMakeLists.txt

```diff
@@ -122,7 +130,6 @@ if(TRAVIS_CI_BUILD)
 endif()
 
 if(CMAKE_COMPILER_IS_GNUCC)
-  include(CheckCCompilerFlag)
   check_c_compiler_flag(-Og HAS_OG_FLAG)
 else()
   set(HAS_OG_FLAG 0)
```

Comment:
@jszakmeister Is there an intentional reason to wrap this `check_c_compiler_flag()` check here with a `if(CMAKE_COMPILER_IS_GNUCC)`?